sources/Win32/Win32/DirectX/d3d12/d3dcommon/D3D_CBUFFER_TYPE.gen.cs

```diff
@@ -4,16 +4,46 @@
 #pragma warning disable CS1589, CS0419, CA1416, CS0618
 namespace Silk.NET.DirectX;
 
+/// <include file='D3D_CBUFFER_TYPE.xml' path='doc/member[@name="D3D_CBUFFER_TYPE"]/*'/>
 public enum D3D_CBUFFER_TYPE
 {
+    /// <include file='D3D_CBUFFER_TYPE.xml' path='doc/member[@name="D3D_CBUFFER_TYPE.D3D_CT_CBUFFER"]/*'/>
+
     D3D_CT_CBUFFER = 0,
+
+    /// <include file='D3D_CBUFFER_TYPE.xml' path='doc/member[@name="D3D_CBUFFER_TYPE.D3D_CT_TBUFFER"]/*'/>
+
     D3D_CT_TBUFFER = (D3D_CT_CBUFFER + 1),
+
+    /// <include file='D3D_CBUFFER_TYPE.xml' path='doc/member[@name="D3D_CBUFFER_TYPE.D3D_CT_INTERFACE_POINTERS"]/*'/>
+
     D3D_CT_INTERFACE_POINTERS = (D3D_CT_TBUFFER + 1),
+
+    /// <include file='D3D_CBUFFER_TYPE.xml' path='doc/member[@name="D3D_CBUFFER_TYPE.D3D_CT_RESOURCE_BIND_INFO"]/*'/>
+
     D3D_CT_RESOURCE_BIND_INFO = (D3D_CT_INTERFACE_POINTERS + 1),
+
+    /// <include file='D3D_CBUFFER_TYPE.xml' path='doc/member[@name="D3D_CBUFFER_TYPE.D3D10_CT_CBUFFER"]/*'/>
+
     D3D10_CT_CBUFFER = D3D_CT_CBUFFER,
+
+    /// <include file='D3D_CBUFFER_TYPE.xml' path='doc/member[@name="D3D_CBUFFER_TYPE.D3D10_CT_TBUFFER"]/*'/>
+
     D3D10_CT_TBUFFER = D3D_CT_TBUFFER,
+
+    /// <include file='D3D_CBUFFER_TYPE.xml' path='doc/member[@name="D3D_CBUFFER_TYPE.D3D11_CT_CBUFFER"]/*'/>
+
     D3D11_CT_CBUFFER = D3D_CT_CBUFFER,
+
+    /// <include file='D3D_CBUFFER_TYPE.xml' path='doc/member[@name="D3D_CBUFFER_TYPE.D3D11_CT_TBUFFER"]/*'/>
+
     D3D11_CT_TBUFFER = D3D_CT_TBUFFER,
+
+    /// <include file='D3D_CBUFFER_TYPE.xml' path='doc/member[@name="D3D_CBUFFER_TYPE.D3D11_CT_INTERFACE_POINTERS"]/*'/>
```

Comment:
Is it intentional that these are being generated again? (Assume we don't care for now and we're just suppressing the warning until we have a proper autodoc?)

(I know I asked for the commented out line to be something else other than a commented out line, but equally I assume this was done for a reason, but feel free to dismiss if this'll be dealt with later)

Reply:
Yeah it is intentional. I want to handle the autodoc later (possibly a separate PR) and for now it is just getting suppressed. I forgot I had the line commented out to check getting proper builds, but the intention is to have that there since even if we don't want these types of docs I do want the inheritdoc generated which is part of the same flag in clangsharp.